definition-map.js

```diff
@@ -75,7 +75,56 @@ const QUERIES = {
             includeCorrectAnswers: "Flag to include questions the user previously got correct.",
             quizzes: "The IDs of the Quizzes user selects."
         }
-    }
+    },
+    Forums: {
+        metadescription: "This query returns a forum object. A forum is any area you can add a discussion to. For example, assignments, forum page type, discussions below the course on each section.",
```

Comment:
Nitpicky, but the wording here makes it sound like you add the discussion to the forum vs the forum being where the discussion takes place. Would probably say something like "returns a forum containing discussion threads within a course, discussion widget, or assignment."